### DISCUSSION TITLE
Email addresses with non-latin result in error

### DISCUSSION THREAD

**Discussion by coder66** [PHPMailer/PHPMailer#2906] [Q&A] [answered]
When we have email addresses with non latin characters like +, ö, ä, ü, ß and so on, there is an error message:

 [error] [application] MessagingHelper::sendMailBatch: 207: Email sending error for invoice dgSA2IY20cZ5D1A6: You must provide at least one recipient email address.

Any idea why this is failing and what setting could fix this? Thanks!

**Comment by Synchro** [comment]
It depends where in the address those characters appear. PHPMailer supports international domain names (IDN), so you can have `user@éxample.com`, however, PHPMailer doesn't support the SMTPUTF8 SMTP extension, so you can't have non-ASCII characters in the local part of the address, for example `üser@example.com`. One reason for this is that (unlike IDN) it requires that the server supports it, and many do not. If you have provided an address like that and the server doesn't do SMTPUTF8, all it can reasonably do is fail to send altogether, because the downgrade of addresses is even less supported.

I'd certainly like to add this capability, but there are many other things I'd like to deal with first, such as consistent header handling and arbitrary MIME structures. For now, if you really need this, I'd recommend using another library that does support SMTPUTF8, like Symfony Mailer or Zend_Mail.

**Comment by coder66** [comment]
Thanks for the explanation. Our hosting provider told us, they are not able to activate SMTPUTF8 server side, which is a bummer for us. How would Symfony Mailer or Zend_Mail fix this, if the SMTP server can not handle UFT8?

**Reply by Synchro** [reply, marked as answer]
They wouldn't. Your server has to support SMTPUTF8 if you want non-ASCII characters in local parts.
